client/network/src/protocol/notifications/upgrade/notifications.rs

```diff
@@ -203,13 +196,13 @@ impl<TSubstream> NotificationsInSubstream<TSubstream>
 where
 	TSubstream: AsyncRead + AsyncWrite + Unpin,
 {
-	#[cfg(test)]
-	pub fn new(
-		socket: Framed<TSubstream, UviBytes<io::Cursor<Vec<u8>>>>,
-		handshake: NotificationsInSubstreamHandshake,
-	) -> Self {
-		Self { socket, handshake }
-	}
+	// #[cfg(test)]
+	// pub fn new(
+	// 	socket: Framed<TSubstream, UviBytes<io::Cursor<Vec<u8>>>>,
+	// 	handshake: NotificationsInSubstreamHandshake,
+	// ) -> Self {
+	// 	Self { socket, handshake }
+	// }
```

Comment:
leftover?

Reply:
no, I had to comment a bunch of tests in `handler.rs`, which were using this API